src/main/java/org/broadinstitute/hellbender/tools/walkers/conversion/CompareGtfInfo.java

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.hellbender.tools.walkers.conversion;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.util.Interval;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.util.Comparator;
+import java.util.Map;
+
+public class CompareGtfInfo implements Comparator<Map.Entry<String, GtfInfo>> {
+
+    private final SAMSequenceDictionary dictionary;
+
+    CompareGtfInfo(SAMSequenceDictionary dictionary) {
+        this.dictionary = dictionary;
+    }
+
+    // compare two entries of a map where key = geneId or transcriptId and value = gtfInfo object
+    @Override
+    public int compare(Map.Entry<String, GtfInfo> e1, Map.Entry<String, GtfInfo> e2) {
```

Comment:
This style of comparator is fine, but if you don't know about it already I might check out `Comparator.comparing(x).thenComparing(y)` style.  I find it much easier to reason about.